### PR TITLE
remove @types/isomorphic-fetch and fix compile issue

### PR DESCRIPTION
**Pull request by islishude**
```
> solidity@0.0.124 compile /project/github.com/juanfranblanco/vscode-solidity
> tsc -p ./

node_modules/@types/isomorphic-fetch/index.d.ts:8:30 - error TS2552: Cannot find name 'fetch'. Did you mean '_fetch'?

8 declare const _fetch: typeof fetch;
                               ~~~~~

  node_modules/@types/isomorphic-fetch/index.d.ts:8:15
    8 declare const _fetch: typeof fetch;
                    ~~~~~~
    '_fetch' is declared here.


Found 1 error.
```

### CODE REVIEW

**Comment by juanfranblanco** [comment]
This does not run in a browser, why the DOM?

**Comment by islishude** [comment]
I can't compile it, and add `dom` to lib in tsconfig, then it works.

```console
$ npx tsc                 
node_modules/@types/isomorphic-fetch/index.d.ts:8:30 - error TS2552: Cannot find name 'fetch'. Did you mean '_fetch'?

8 declare const _fetch: typeof fetch;
                               ~~~~~

  node_modules/@types/isomorphic-fetch/index.d.ts:8:15
    8 declare const _fetch: typeof fetch;
                    ~~~~~~
    '_fetch' is declared here.


Found 1 error.
```

**Comment by islishude** [comment]
I just found that the `@types/isomorphic-fetch` is not necessary for this project, so I think we can remove this dependency.

**Comment by juanfranblanco** [comment]
I don't have any issues in a new build. Have you upgraded some packages?

**Comment by islishude** [comment]
> I don't have any issues in a new build. Have you upgraded some packages?

No,I start from a clean workspace

**Comment by islishude** [comment]
just removed `@types/isomorphic-fetch`, it can build now.

and also removed a section of build issue in README-DEVELOP.md

**Comment by juanfranblanco** [comment]
Oh!! That is strange. What OS are you using?

**Comment by islishude** [comment]
> Oh!! That is strange. What OS are you using?

MacOS

File: README-DEVELOP.md [hunk 1]
```diff
@@ -22,18 +21,6 @@ You need to have installed
...
-
-## Dependency hell / Current Issues
-There is a dependency on isomorphic-fetch
-if having issues compiling, replace the index.d.ts file with this
-
-```typescript
-declare namespace _fetch { }
-declare const _fetch: any;
-export = _fetch;
-```
```
Comment (islishude) [inline README-DEVELOP.md:34]: @juanfranblanco I don't add those codes. as I said before, `isomorphic-fetch` can be removed

**Comment by juanfranblanco** [comment]
Ok I have tested it in windows and linux too, it was a conflict at the time, but it seems to be ok now.

**Pull request merged**